hypertrace-ingester/src/test/java/org/hypertrace/migration/MigrationTestHttp.java

```diff
@@ -44,21 +33,17 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 import org.hypertrace.core.datamodel.RawSpan;
-import org.hypertrace.core.semantic.convention.constants.error.OTelErrorSemanticConventions;
 import org.hypertrace.core.semantic.convention.constants.http.OTelHttpSemanticConventions;
-import org.hypertrace.core.semantic.convention.constants.rpc.OTelRpcSemanticConventions;
 import org.hypertrace.core.semantic.convention.constants.span.OTelSpanSemanticConventions;
 import org.hypertrace.core.span.constants.RawSpanConstants;
 import org.hypertrace.core.spannormalizer.jaeger.JaegerSpanNormalizer;
 import org.hypertrace.semantic.convention.utils.http.HttpSemanticConventionUtils;
-import org.hypertrace.semantic.convention.utils.rpc.RpcSemanticConventionUtils;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.Arguments;
 import org.junit.jupiter.params.provider.MethodSource;
 
-public class MigrationTest {
+public class MigrationTestHttp {
```

Comment:
status code related attribute is missing from tagMap - see testHttpFields. Let's add one.
```
RawSpanConstants.getValue(OT_SPAN_TAG_HTTP_STATUS_CODE),
RawSpanConstants.getValue(HTTP_RESPONSE_STATUS_CODE),
OTelHttpSemanticConventions.HTTP_STATUS_CODE.getValue()
```